Kryptonite/Silo.swift

```diff
@@ -140,18 +139,18 @@ class Silo {
             }
 
         case .git(let gitSign):
-            Analytics.postEvent(category: request.body.analyticsCategory, action: "automatic approval", label: communicationMedium.rawValue)
-            
             if let error = gitSign.error {
                 Policy.notifyUser(errorMessage: error, session: session)
             } else {
                 Policy.notifyUser(session: session, request: request)
             }
 
-        case .me, .ack, .unpair:
+        case .me, .ack, .unpair, .hosts:
             break
         }
         
+        Analytics.postEvent(category: request.body.analyticsCategory, action: "automatic approval", label: communicationMedium.rawValue)
```

Comment:
this will send analytics for me, ack, and unpair requests which I think won't really be useful. what about limiting it to just `.hosts` and the ones we currently send it for?

Reply:
👍 yes, i moved from the individual cases not realizing this.